docs/en/setup/backend/backend-alarm.md

```diff
@@ -180,6 +180,23 @@ slackHooks:
     - https://hooks.slack.com/services/x/y/z
 ```
 
+## WeChat Hook
+To use Wechat webhook you need to follow the [Wechat Webhooks guide](https://work.weixin.qq.com/api/doc/90000/90136/91770).
```

Comment:
According to this doc, it seems to belong to Enterprise WeChat, could you confirm? Where is the doc about how to get a group robot?

Reply:
Yes, this is the Enterprise wechat, I will change the link later

Reply:
No, this can't be done in a normal WeChat group?

Reply:
Yes, there is not open api for normal WeChat group, it's not illegal

Reply:
Then let's indicate it clearly in the document. Because people may be misguided by the current doc. The first look to me is, this works on any WeChat until I open the API doc you linked.

Reply:
got it